src/iidm/ConfiguredBus.hpp

```diff
@@ -25,7 +24,7 @@ class BusTerminal;
 class Network;
 class Terminal;
 
-class ConfiguredBus : public Bus, public MultiVariantObject {
+class ConfiguredBus : public Bus {
```

Comment:
To be checked: does the implementation of MultiVariantObject methods ok (super call?)

Reply:
Ok, done by backporting Java PR #1476